config.yml

```diff
@@ -1,6 +1,13 @@
 version: 1
 build:
   main: cmd/poktrolld
+  ldflags:
+    # The dao reward address SHOULD match that of the "pnf" below (i.e. `make poktrolld_addr ACC_NAME=pnf`).
```

Comment:
Why is this a SHOULD and not a MUST?

Reply:
My rationale was that if this is "wrong" nothing is going to break, the settlement rewards will just be in a different account.

Reply:
This comment has moved to `genesis.tokenomics.params`:

```yaml
    tokenomics:
      params:
        mint_allocation_percentages:
        # ... other params
        # The dao reward address SHOULD match that of the "pnf" below (i.e. `make poktrolld_addr ACC_NAME=pnf`).
        # This is the address that will receive the dao/foundation rewards during claim settlement (global mint TLM).
        dao_reward_address: "pokt1eeeksh2tvkh7wzmfrljnhw4wrhs55lcuvmekkw"
```